.github/workflows/component-builder.yml

```diff
@@ -0,0 +1,59 @@
+name: "Component Builder"
+on:
+  pull_request_target:
+    paths:
+      - 'containers/fedora/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  guest-fedora-amd64:
+    runs-on: ubuntu-latest
+    env:
+      FEDORA_IMAGE: Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2
```

Comment:
I think that the resource for building an image shouldn't be hard-coded and limited to a specific version (41).
You probably should take it from [fedora_os_matrix](https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/global_config.py#L363) (via [FEDORA_LATEST](https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/os_params.py#L27)).

Reply:
But you do set `FEDORA_VERSION` below in line 15. So why not setting it before, and use it dynamically?
```
FEDORA_IMAGE: Fedora-Cloud-Base-Generic-${{ env.FEDORA_IMAGE }}-1.4.x86_64.qcow2
```
or something like that?

Reply:
The line you commented on is the source of the Fedora cloud image.
The `FEDORA_VERSION` below is used to name the different files and tags.

They could be linked, but it is the other way around: Parse the image name from the URL provided to the `FEDORA_IMAGE`, extract the major version and use it as the `FEDORA_VERSION`.

Reply:
OK, I think I see what you mean here.
Thanks for the explanation.